docs/compass/03-01-security.md

```diff
@@ -217,3 +218,32 @@ foo@bar.com:
 **Scopes**
 
 Scopes are added to the authentication session in Tenant Mapping Handler. The handler gets not only `tenant`, but also `scopes`, which are fixed regarding of type of the object (Application / Runtime). Application and Runtime are always have the same scopes defined.
+
+### One-time token
+
+**Used by:** Runtime/Application
+
+**Connector flow:**
+
+1. Runtime/Application makes a call to the Connector's internal API.
+1. The OathKeeper uses the Token Resolver as a mutator. 
+The Client ID is extracted from the one-time token's `Connector-Token` header or from the `token` query parameter, and is then written to the `Client-Id-From-Token` header. In the case of failure, the header is empty.
+1. The OathKeeper proxies the request further to the Compass Gateway.
+1. The request is forwarded to the Connector.
+
+![Auth](./assets/token-security-diagram-connector.svg)
```

Comment:
I think there is some object/subject missing on the diagram (and on other as well) that initiates step 1.

Reply:
I'll discuss it with @klaudiagrz when she gets back so that all of them are unified.